test/range_distance_compat_test.cpp

```diff
@@ -5,7 +5,7 @@
 // http://www.boost.org/LICENSE_1_0.txt)
 
 #include <boost/range/distance.hpp>
-#include <boost/type_traits/is_integral.hpp>
+#include <boost/type_traits/is_integral.hpp>  // There is a missed include in boost/range/concepts.hpp. This include must be removed in the future.
```

Comment:
Please file a PR for Boost.Range rather than add unnecessary includes like this. I bet noone will remember to remove this include in the future.